CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+- 🎁 The schema definition parser has been improved:
+  - it can now accept references to types that are defined later on in the same
+    file.
+  - type definitions from already parsed schema files can be referenced from
+    files that are parsed later. Each directory from the `schema-dirs` option
+    is traversed in lexicographical order.
+  [#1331](https://github.com/tenzir/vast/pull/1331)
```

Comment:
```suggestion
- ⚠️ Schema parsing now uses a 2-pass loading phase so that type aliases can
  reference other types that are later defined in the same file. Additionally,
  type definitions from already parsed schema files can be referenced from
  files that are parsed later. The directories from the `schema-dirs` option
  are parsed exactly as specified and traversed in lexicographical order.
  [#1331](https://github.com/tenzir/vast/pull/1331)
```

Reply:
I changed the text a little more to reflect the newly agreed upon semantics.